go.mod

```diff
@@ -37,4 +37,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```

Comment:
Reminder.

Reply:
addressed in [b6bbe07](https://github.com/codeready-toolchain/registration-service/pull/260/commits/b6bbe07febe25b04300a25ed9d4b811bfe29c073)